autorank/autorank.py

```diff
@@ -111,6 +111,9 @@ def autorank(data, alpha=0.05, verbose=False, order='descending', approach='freq
         automatically get the t-test/repeated measures ANOVA. With 'nonparametric' you automatically get Wilcoxon's
         signed rank test/Friedman test.
 
+    random_state (integer, default=None):
+        Seed for random state. Can be used to compute reproducible results.
```

Comment:
Please modify the second sentence to "Forwarded to Bayesian signed rank test to enable reproducible sampling and, thereby, reproducible results."